chrome/content/zotfile/zotfile.js

```diff
@@ -1926,7 +1926,14 @@ Zotero.ZotFile = {
               // create a nslFile Object of the destination folder
               var dir = this.createFile(destination);
               // move file to new location
-              file.moveTo(dir, filename);
+              if(destination.trim() == "/") { 
```

Comment:
If user doesn't specify anything in the box `Zotfile Preferences > General Settings > Location of files > Custom Location`, then it's `/`. I don't know for Mac and Windows. Can you please check? If the field is empty then, we can also set the destination in initialization (in the function `getLocation`) to something what we can recognize later. What's your take on this?